app/models/profile.rb

```diff
@@ -1,4 +1,12 @@
 class Profile < ApplicationRecord
   belongs_to :user
   has_many :ingestions
```

Comment:
Naming
I suggest to use `meals` instead of `ingestions`

Reply:
OK, so we've discussed this online and with currect architecture there is no other options right now. Will be later changed after architecture review.